EIPS/eip-5625.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 5625
+title: NFT Metadata JSON Schema dStorage Extension
+description: Add a dStorage property to non-fungible tokens (NFTs) metadata JSON schema to provide decentralized storage information of NFT assets
+author: Gavin Fu (@gavfu)
+discussions-to: https://ethereum-magicians.org/t/eip-5625-nft-metadata-json-schema-dstorage-extension/10754
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-08
+requires: 721, 1155
+---
+
+## Abstract
+
+This standard is an extension to NFT metadata JSON schema defined by [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md). As a complement, it provides additional information about how the NFT asset is stored in a decentralized storage system, if applicable.
+
+## Motivation
+
+As high valuable crypto property, NFT assets intrinsically demand guaranteed storage to assure their **immutability**, **reliability** and **durability**. NFT ownership is tracked by [EIP-721](./eip-721.md) or [EIP-1155](./eip-1155.md) smart contracts, hence persisted in blockchain, which is not a problem. But how about the mime type assets that NFT tokens actually represent? Ideally, they should also be stored in some reliable and verifiable decentralized storage system which is designed to store larger amounts of data than the blockchain itself. As an effort to promote **decentralized storage** adoption in NFT world, we propose to add additional **dStorage** information into NFT metadata JSON schema.
+
+As a refresher, let's review existing NFT metadata JSON schema standards. [EIP-721](./eip-721.md) defines a standard contract method `tokenURI` to return a given NFT's metadata JSON file, conforming to the *[EIP-721](./eip-721.md) Metadata JSON Schema*, which defines three properties: `name`, `description` and `image`.
+
+Similarly, [EIP-1155](./eip-1155.md) also defines a standard contract method `uri` to return NFT metadata JSON files conforming to the *[EIP-1155](./eip-1155.md) Metadata JSON Schema*, which defines properties like `name`, `decimals`, `description`, `image`, `properties`, `localization`, etc.
+
+Besides, as the world's largest NFT marketplace nowadays, OpenSea defines their own *Metadata Standards*, including a few more properties like `image_data`, `external_url`, `attributes`, `background_color`, `animation_url`, `youtube_url`, etc. This standard is de facto respected and followed by other NFT marketplaces like LooksRare.
+
+Apparently, none of these standards conveys storage information about the mime type asset that NFT token actually represents. This proposal is an effort to fill the missing part.
```

Comment:
I spot a few grammar issues here. If you run this through something like Grammarly and fix the issues, I won't have to nitpick this and your EIP will be merged much quicker.

Reply:
Really appreciate your review. This section is checked via Grammarly and grammar issues are fixed accordingly. I'm not a native English speaker or writer, so if there are any more grammar issues to fix, or some place to rephrase, please don't hesitate to point out.  :)